src/Makefile.am

```diff
@@ -4,15 +4,16 @@ RIGSRC = rig.c serial.c serial.h misc.c misc.h register.c register.h event.c \
 	event.h cal.c cal.h conf.c tones.c tones.h rotator.c locator.c rot_reg.c \
 	rot_conf.c rot_conf.h iofunc.c iofunc.h ext.c mem.c settings.c \
 	parallel.c parallel.h usb_port.c usb_port.h debug.c network.c network.h \
-	cm108.c cm108.h gpio.c gpio.h idx_builtin.h token.h par_nt.h microham.c microham.h
+	cm108.c cm108.h gpio.c gpio.h idx_builtin.h token.h par_nt.h microham.c microham.h \
+  amplifier.c amp_reg.c amp_conf.c amp_conf.h extamp.c
```

Comment:
Formatting issues. I wonder if we should simply do a separate PR that fixes formatting for all files before 4.0 release? We do have clearly documented rules for formatting anyway.